presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -43,7 +43,7 @@ Mathematical Functions
 .. function:: chi_squared_cdf(df, value) -> double
 
     Compute the Chi-square cdf with given df (degrees of freedom) parameter:  P(N < value; df).
-    The df parameter must be positive real numbers and value must be a real value.
+    The df parameter must be a positive real number (of type DOUBLE) and value must be a real value.
```

Comment:
Value is also a DOUBLE right?

Reply:
Thanks, done.

Reply:
Hmm, I still see that the type isn't mentioned for `value`.  Isn't this also significant?

Reply:
Oops, I thought you meant the value of df is DOUBLE (which was already implemented). Yes, the type of 'value' is also DOUBLE. I'll fix it now.

Reply:
NOW it's fixed.